modules/apps/27-interchain-accounts/controller/keeper/keeper.go

```diff
@@ -206,3 +206,21 @@ func (k Keeper) SetInterchainAccountAddress(ctx sdk.Context, connectionID, portI
 	store := ctx.KVStore(k.storeKey)
 	store.Set(icatypes.KeyOwnerAccount(portID, connectionID), []byte(address))
 }
+
+// IsMiddlewareEnabled returns true if the underlying application calbacks are enabled for given port and channel identifier pair, otherwise false
```

Comment:
```suggestion
// IsMiddlewareEnabled returns true if the underlying application callbacks are enabled for given port and channel identifier pair, otherwise false
```